test/recipes/30-test_evp_data/evppkey_rsa_common.txt

```diff
@@ -969,12 +970,14 @@ Input="0123456789ABCDEF0123"
 Output = 3EFE09D88509027D837BFA5F8471CF7B69E6DF395DD999BB9CA42021F15722D9AC76670507C6BCFB73F64FB2211B611B8F140E76EBDB064BD762FDBA89D019E304A0D6B274E1C2FE1DF50005598A0306AF805416094E2A5BA60BC72BDE38CE061E853ED40F14967A8B9CA4DC739B462F89558F12FDF2D8D19FBEF16AD66FE2DDDA8BEE983ECBD873064244849D8D94B5B33F45E076871A47ED653E73257A2BE2DB3C0878094B0D2B6B682C8007DFD989425FB39A1FEEC9EED5876414601A49176EC344F5E3EDEE81CA2DDD29B7364F4638112CB3A547E2BC170E28CB66BDABE863754BE8AD5BA230567B575266F4B6B4CF81F28310ABF05351CC9E2DB85D00BF
 
 # Verify using salt length larger than minimum
```

Comment:
If it makes sense Availablein = default is ok also.

Reply:
Since @paulidale mentions `Not checking against the default provider if the FIPSversion is specified is only temporary.` [here](https://github.com/openssl/openssl/pull/23900#discussion_r1697815912), I think it is better to use `FIPSversion` instead of `Availablein`  to test it with default provider and FIPS provider prior to 3.4.0.